tests/cypress/e2e/features/consensus.js

```diff
@@ -64,20 +65,161 @@ context('Basic manipulations with consensus job replicas', () => {
+        const attrName = labelName;
+        const advancedConfigurationParams = { consensusReplicas: replicas };
+
+        const defaultArgs1 = ['Some default value for type Text', '', null];
+        const deafultArgs2 = [false, false, '', 'success', 'Test', null];
+
+        before(() => {
+            cy.goToTaskList();
+            cy.createAnnotationTask(
+                taskName, labelName, attrName,
+                ...defaultArgs1,
+                advancedConfigurationParams,
+                ...deafultArgs2,
+                serverFiles,
+            ); // TODO: rewrite to headless call once the task consensus endpoint is finished
+            cy.openTask(taskName);
+            cy.get('.cvat-consensus-job-collapse').click();
+        });
+        it("Check new merge buttons exist and are visible. Trying to merge 'new' jobs should trigger errors", () => {
+            // Merge all consensus jobs in task
+            cy.mergeConsensusTask(400);
+            cy.get('.cvat-notification-notice-consensus-merge-task-failed')
+                .should('be.visible')
+                .invoke('text')
+                .should('include', 'Could not merge the task');
+            cy.closeNotification('.cvat-notification-notice-consensus-merge-task-failed');
+
+            // Merge one consensus job
+            cy.mergeConsensusJob(400);
+            cy.get('.cvat-notification-notice-consensus-merge-task-failed')
+                .should('be.visible')
+                .invoke('text').should('include', 'Could not merge the job');
+            cy.closeNotification('.cvat-notification-notice-consensus-merge-task-failed');
 
-            // Check asc order of jobs
+            // Check asc order of jobs and save job ids
             function parseJobId(jobItem) {
                 const jobItemText = jobItem.innerText;
                 const [start, stop] = [0, jobItemText.indexOf('\n')];
                 return +(jobItemText.substring(start, stop).split('#')[1]);
             }
             cy.get('.cvat-job-item').then((jobItems) => {
                 const sourceJobId = parseJobId(jobItems[0]);
+                jobIDs.push(sourceJobId);
                 for (let i = 1; i <= replicas; i++) {
                     const jobId = parseJobId(jobItems[i]);
+                    jobIDs.push(jobId);
                     expect(jobId).equals(sourceJobId + i);
                 }
             });
         });
+
+        it('Check consensus management page', () => {
+            const defaultQuorum = 50;
+            const defaultIoU = 40;
+            cy.contains('button', 'Actions').click();
+            cy.contains('Consensus management').should('be.visible').click();
+            cy.get('.cvat-consensus-management-inner').should('be.visible');
+            // Save settings, confirm request is sent
+            let requestCount = 0;
+            cy.intercept('PATCH', 'api/consensus/settings/**', () => {
+                requestCount++;
```

Comment:
I dont think we need to check request count. By default its not supposed to be sent. There was no regressions where it was sent.